### PR TITLE
Update Set-NetTCPSetting.md

### PR DESCRIPTION
**Pull request by sowjanya365**
Issue - "fill in the blank" assignments - Description has been updated now.

### CODE REVIEW

**Comment by e0i** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

**Comment by e0i** [comment]
@dariomws306 @get-itips @tseward

Peer review needed for this PR. Thanks.

**Comment by dariomws306** [comment]
Same as:
https://github.com/MicrosoftDocs/windows-powershell-docs/pull/1322

**Comment by tseward** [comment]
Agree with @dariomws306

**Comment by sowjanya365** [comment]
Hi @dariomws306 @tseward 

I've updated the document with the suggested modifications. Kindly review. Thank you.

File: docset/winserver2012r2-ps/nettcpip/Set-NetTCPSetting.md [hunk 1]
```diff
@@ -101,7 +103,9 @@ Accept wildcard characters: False
 ```
 
 ### -AutoReusePortRangeStartPort
-{{Fill AutoReusePortRangeStartPort Description}}
+Specifies the number of ports for the auto-reuse port range, which is a port range used for local ephemeral port selection by outbound TCP connections for which either SO_REUSE_UNICASTPORT has been set on the socket, or for which connect() has been called without calling bind() on the socket.
+
+If you specify 0, the auto-reuse feature is disabled and ephemeral ports are drawn instead from the dynamic port range as specified by *DynamicPortRangeStartPort* and *DynamicPortRangeNumberOfPorts*, even if SO_REUSE_UNICASTPORT is set on a socket.
```
Comment (JohanFreelancer9) [inline docset/winserver2012r2-ps/nettcpip/Set-NetTCPSetting.md]:
```suggestion
If you specify 0, the auto-reuse feature is disabled and ephemeral ports are drawn instead from the dynamic port range as specified by DynamicPortRangeStartPort and DynamicPortRangeNumberOfPorts, even if SO_REUSE_UNICASTPORT is set on a socket.
```

File: docset/winserver2012r2-ps/nettcpip/Set-NetTCPSetting.md [hunk 2]
```diff
@@ -86,7 +86,9 @@ Accept wildcard characters: False
 ```
 
 ### -AutoReusePortRangeNumberOfPorts
-{{Fill AutoReusePortRangeNumberOfPorts Description}}
+Specifies the number of ports for the auto-reuse port range, which is a port range used for local ephemeral port selection by outbound TCP connections for which either SO_REUSE_UNICASTPORT has been set on the socket, or for which connect() has been called without calling bind() on the socket.
+
+If you specify 0, the auto-reuse feature is disabled and ephemeral ports are drawn instead from the dynamic port range as specified by *DynamicPortRangeStartPort* and *DynamicPortRangeNumberOfPorts*, even if SO_REUSE_UNICASTPORT is set on a socket.
```
Comment (JohanFreelancer9) [inline docset/winserver2012r2-ps/nettcpip/Set-NetTCPSetting.md]:
```suggestion
If you specify 0, the auto-reuse feature is disabled and ephemeral ports are drawn instead from the dynamic port range as specified by DynamicPortRangeStartPort and DynamicPortRangeNumberOfPorts, even if SO_REUSE_UNICASTPORT is set on a socket.
```

**Comment by dariomws306** [comment]
Looks good now.

Good work!

Thank you!

**Pull request merged**